contracts/.gas-snapshot

```diff
@@ -0,0 +1,52 @@
+BurnMintERC677_approve:testApproveSuccess() (gas: 55512)
```

Comment:
Please delete, this is not the correct snapshot location. Any time you see an unexpected new file please ensure it was intended to create it